build-system/tasks/karma.conf.js

```diff
@@ -69,6 +78,19 @@ module.exports = {
     transform: [['babelify', {caller: {name: 'test'}, global: true}]],
     // Prevent "cannot find module" errors on Travis. See #14166.
     bundleDelay: isTravisBuild() ? 5000 : 1200,
+
+    persistentCache: browserifyPersistFs(
+      './build/.karma-cache',
+      {
+        deps: createHash(fs.readFileSync('./yarn.lock')),
+        build: globby.sync('build-system/**/*.js').map((f) => {
+          return createHash(fs.readFileSync(f));
+        }),
+      },
+      () => {
+        process.stdout.write('.');
+      }
```

Comment:
WDYT about adding a `maxAge` value (say, 1 week?) to enable [garbage collection](https://github.com/martinheidegger/browserify-persist-fs#garbage-collection) for the cache?

Reply:
Done.